docs/reference/extensions/flask-framework.rst

```diff
@@ -41,6 +47,23 @@ application. In the following example we use it to specify ``libpq-dev``:
         # list required packages or slices for your flask app below.
         - libpq-dev
 
+.. _flask-async-gunicorn-workers:
+
+Using asynchronous Gunicorn workers
+===================================
+
+If you want to use asynchronous workers, you have to add ``gevent`` package to
+the ``requirements.txt`` file. Rockcraft automatically detects this and updates
+the pebble plan to use the asynchronous workers. If you have ``gevent``
+installed in your rock but decided to use ``sync`` workers instead you can use
+the ``--args`` parameter of ``docker run`` to use ``sync`` workers instead of
+the default ``gevent``:
```

Comment:
I think we can restyle this section:

- Since this is a reference, we generally want to frame this more as mechanical explantions than how-to language.
- I found a few ways to tighten up the language and clarify meaning.

```suggestion
Gunicorn worker selection
=========================

If the project has gevent as a dependency, Rockcraft automatically updates the
pebble plan to spawn asynchronous Gunicorn workers.

When the project instead needs synchronous workers, you can override the worker
type by adding ``--args flask sync`` to the Docker command that launches the
rock:
```